oms-monitor/docker/docker-compose.yml

```diff
@@ -94,6 +94,9 @@ services:
       - "9600:9600"
     environment:
       LS_JAVA_OPTS: "-Xmx256m -Xms256m"
+      ELASTIC_HOST: "${ELASTIC_HOST}"
+      ELASTIC_USER: "${ELASTIC_USER}"
+      ELASTIC_PASSWORD: "${ELASTIC_PASSWORD}"
```

Comment:
really? does it work this way?

Reply:
yep, this is on b1 right now and it works